jwst/cube_build/ifu_cube.py

```diff
@@ -255,8 +261,30 @@ def set_geometry(self, corner_a, corner_b, lambda_min, lambda_max):
         # we have angles in degrees
         ra_ave = circmean(ravalues * u.deg).value
 
-        self.crval1 = ra_ave
-        self.crval2 = dec_ave
+        if self.ra_center is not None:
+            # check that user provided ra center are within the already
+            # determined min and max values. 
```

Comment:
Do we need to require this?  I'm thinking ahead to some programs that tile larger galaxies, and the desired cube center could be outside the footprint of any individual observation.  I think the consequence would be just that the cubes get absurdly big right?

Reply:
Ah ok. It was just a check for users not entering incorrect values. But I had not thought of that case. I will remove it